### PR TITLE
[DO NOT MERGE] Fix apply chunks

### PR DESCRIPTION
**Pull request by bowenwang1996**
Submit a PR so that it is easier to see what has changed

### CODE REVIEW

File: runtime/runtime/src/lib.rs [hunk 1]
```diff
@@ -1120,6 +1125,23 @@ impl Runtime {
                 &mut stats,
             )?;
         }
+        if !apply_state.is_new_chunk
+            && apply_state.current_protocol_version
+                >= PROTOCOL_FEATURES_TO_VERSION_MAPPING[&ProtocolFeature::FixApplyChunks]
+        {
```
Comment (frol) [inline runtime/runtime/src/lib.rs:1131]: Should we use `checked_feature!`?

**Review by frol — APPROVED** [review]
I only reviewed it on a high level. LGTM.

File: runtime/runtime/src/lib.rs (hunk 1 above)
Comment (bowenwang1996) [reply runtime/runtime/src/lib.rs:1131]: checked_feature does not work when there is no associated rust feature

**Comment by frol** [comment]
Already released as 1.18.3: https://github.com/near/nearcore/compare/1.18.2...1.18.3

**Pull request closed without merging**